docs/user-guide/release-notes/2.6.x.rst

```diff
@@ -47,6 +47,20 @@ Deprecation
    python-gofer-amqplib with python-gofer-amqp if installed.
 
 
+.. _2.4.x_upgrade_to_2.6.0:
+
+Upgrade Instructions for 2.4.x --> 2.6.0
```

Comment:
We don't normally give upgrade instructions that skip releases. Did you mean 2.5.<latest?>?